http/transport/circuit_breaker_tripper_test.go

```diff
@@ -0,0 +1,64 @@
+package transport
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/sony/gobreaker"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCircuitBreakerTripper(t *testing.T) {
+	req := httptest.NewRequest("GET", "/foo", nil)
+
+	t.Run("with_default_settings", func(t *testing.T) {
+		breaker := NewDefaultCircuitBreakerTripper("testcircuitbreaker")
+		chain := Chain(breaker).Final(&failingRoundTripper{})
+
+		for i := 0; i < 6; i++ {
```

Comment:
Why is it 6 instead of 5? I thought it would open after the 5th failed request, so the 6th should already be returning the open state error. What am I missing?

Reply:
Because the rule is `counts.ConsecutiveFailures > 5` so he is opening after the 5th (with the 6th) 
failed consecutive failed request. The comment might be not clear enough, I will adapt it.